contracts/src/RichardsGame/Pool.sol

```diff
@@ -0,0 +1,173 @@
+/**
+ * @notice This is a gambling pool, which lets you gamble with ERC20s and NFTs, similar to csgoempire.co
+ * 1. Users deposit NFTs and ERC20s into the pool and receive a proportional amount of point
+ * 2. Every hour, the pool is spun, a random number between 0 and the total amount of points is generated
+ * 3. The user who has the point wins everything in the pool, you get 30 seconds to claim otherwise rewards stay in the pool
```

Comment:
30 seconds seems relatively short -- people often end up missing BeReals even with a 2-minute window. Thoughts on increasing post-claim window to 5 or 10 minutes?

Reply:
You'd be on the website, and the pool is rolled over to the next pot anyway, so I don't think anyone would mind -we could make it longer though, 5 mins would be ok I guess

Reply:
Cool, let's do 5m